Simplenote/Classes/SPMarkdownParser.m

```diff
@@ -50,21 +50,26 @@ + (NSString *)htmlHeader
             "<style media=\"screen\" type=\"text/css\">\n";
     NSString *headerEnd = @"</style></head><body><div class=\"note-detail-markdown\">";
 
-    NSString *css = [NSString stringWithContentsOfURL:[[NSBundle mainBundle] URLForResource:self.cssPath withExtension:nil]
-                                             encoding:NSUTF8StringEncoding error:nil];
+    NSString *css = [SPMarkdownParser loadCSSAtPath:self.cssPath];
```

Comment:
Since this method is static, you don't really need to write the classname: 

`[self loadCSSAtPath:...]` should work as well 🔥